src/main/java/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> storage = new HashMap<>();
+
+    public static Map<String, Integer> getStorage() {
+        return storage;
+    }
+
+    public void setStorage(Map<String, Integer> storage) {
```

Comment:
```suggestion
    public void setFruitQuantity(String fruit, Integer quantity) {
```

you don't need to update whole storage